README.md

```diff
@@ -35,6 +35,7 @@ The `bitcoin_hashes` range dependency effects the MSRV as follows
 
 - `bitcoin_hashes v0.12`: MSRV v1.41.1
 - `bitcoin_hashes v0.13`: MSRV v1.48.0
+- `bitcoin_hashes v0.14`: MSRV v1.56.0
```

Comment:
I haven't verified this claim given that we decided to forgo CI checks for now and manually figuring out the necessary pinning is a pita. It seems benign enough and a reasonable assumption that the MSRV will be dictated by `bitcoin_hashes` and there aren't any further interactions in the code base.